deploy/operator.yaml

```diff
@@ -28,7 +28,9 @@ spec:
             name: healthz
           command:
           - marketplace-operator
-          imagePullPolicy: Always
+          args:
+          - -registryServerImage=quay.io/openshift/origin-operator-registry
+          imagePullPolicy: IfNotPresent
```

Comment:
This change to image pull policy will be removed once we update the image right?

Reply:
I don't follow. This is the correct image. Why would would we change the pull policy?

Reply:
Sorry didn't look at the uri. I had thought that we were using a temp image until Abu's pr was pushed built and available in quay. Looks like that happened.

So what I'm not sure about here is why are we changing the imagepullpolicy? Don't we want to always get the latest when this gets applied?

Reply:
You are right. I will fix that here. I also realize I need to update the upstream manifests. :-(